tree/ntuple/v7/src/RPageStorageFile.cxx

```diff
@@ -158,9 +169,15 @@ ROOT::Experimental::Detail::RPageSinkFile::CommitClusterGroupImpl(unsigned char
    auto szPageListZip = fCompressor->Zip(serializedPageList, length, GetWriteOptions().GetCompression(),
                                          RNTupleCompressor::MakeMemCopyWriter(bufPageListZip.get()));
 
+#ifdef MY_CODE
+    this->fWriter->WritePadding();
+#endif
    RNTupleLocator result;
    result.fBytesOnStorage = szPageListZip;
    result.fPosition = fWriter->WriteBlob(bufPageListZip.get(), szPageListZip, length);
+#ifdef MY_CODE
+    this->fWriter->WritePadding();
+#endif
```

Comment:
I'm not sure we should enforce alignment at this level.  Alternatives:
- Force specific alignment on cluster boundary so that each cluster can be individually shared.
- Or force alignment only at start end end of the data section - so that it can only be shared as a whole.

To be discussed in the future.